layouts/partials/tw/thumb-v-item.html

```diff
@@ -1,3 +1,6 @@
+{{ $showByline := .showByline | default "true" }}
```

Comment:
Make it `default true` with no quotes around `true`.

Reply:
![image](https://user-images.githubusercontent.com/222245/236861463-642bfd4a-25ae-4e8d-8841-87c7d317c263.png)

It looks like it's not working. I think the problem is that the default kicks in even when you explicitly set it to false because it replaces falsey values. To fix it, make the options work the other way around and have `{{ $hideByline := .hideByline | default false }}` instead of .showByline and that should work.

Reply:
Looks like the undefined is passing through correctly now, but I'd still prefer to have `true` with no quotes instead of `"true"` as the default.